extensions/arc/deployment/src/main/resources/dev-ui/qwc-arc-observers.js

```diff
@@ -90,6 +101,33 @@ export class QwcArcObservers extends LitElement {
         }
   }
 
+  _renderFilterBar(){
+        return html`<vaadin-text-field
+                        placeholder="Search"
+                        class="filterBar"
+                        @value-changed="${(e) => {
+                            const searchTerm = (e.detail.value || '').trim();
+                            const matchesTerm = (value) => {
+                                if(value){
+                                    return value.toLowerCase().includes(searchTerm.toLowerCase());
+                                }
+                            }
+                            if(searchTerm && searchTerm.trim()!==''){
```

Comment:
```suggestion
                            if(searchTerm?.trim()){
```
JavaScript will interpret empty strings, null, or undefined as false